### DISCUSSION TITLE
Would it be possible to have elevated and non-elevated PowerShell tabs in the same Windows Terminal window?

### DISCUSSION THREAD

**Discussion by HotCakeX** [microsoft/terminal#15997] [Ideas]
The title

**Comment by zadjii-msft** [comment]
Nope. See: #1032, #5000, #14800, #14304, #12885, #6975, and the many, many other threads on the repo

**Reply by HotCakeX** [reply]
Thank you for gathering all the threads, makes sense in terms of security that it's not a good idea to implement it.
